.github/workflows/build-pkg.yml

```
@@ -25,16 +26,57 @@ jobs:
         id: print-version
         run: |
           echo "version=$(git -C brew describe --tags --always)" >> "$GITHUB_OUTPUT"
+      - name: Install the Apple certificate
+        env:
+          PKG_APPLE_SIGNING_CERTIFICATE_BASE64: ${{ secrets.PKG_APPLE_SIGNING_CERTIFICATE_BASE64 }}
+          PKG_APPLE_SIGNING_CERTIFICATE_PASSWORD: ${{ secrets.PKG_APPLE_SIGNING_CERTIFICATE_PASSWORD }}
+          PKG_KEYCHAIN_PASSWORD: ${{ secrets.PKG_KEYCHAIN_PASSWORD }}
+        run: |
+          # create certificate file from base64 encoded secret
+          CERTIFICATE_PATH="${RUNNER_TEMP}/build_certificate.p12"
+          echo -n "${PKG_APPLE_SIGNING_CERTIFICATE_BASE64}" \
+            | base64 --decode -o "${CERTIFICATE_PATH}"
+
+          # create temporary keychain
+          KEYCHAIN_PATH="${RUNNER_TEMP}/app-signing.keychain-db"
```

Minor nit, seems nicer to set these all in one place.
```suggestion
          PKG_KEYCHAIN_PASSWORD: ${{ secrets.PKG_KEYCHAIN_PASSWORD }}
          CERTIFICATE_PATH: ${{ runner.temp }}/build_certificate.p12
          KEYCHAIN_PATH: ${{ runner.temp }}/app-signing.keychain-db
        run: |
          # create certificate file from base64 encoded secret
          echo -n "${PKG_APPLE_SIGNING_CERTIFICATE_BASE64}" \
            | base64 --decode -o "${CERTIFICATE_PATH}"

          # create temporary keychain
```